MapboxCoreNavigationTests/Fixtures/routing/translations/zh-Hans.json

```diff
@@ -0,0 +1 @@
}"},"slight left":{"default":"稍向左下匝道","name":"稍向左下匝道，驶入{way_name}","destination":"稍向左下匝道，前往{destination}","exit":"从左侧{exit}出口驶出","exit_destination":"从左侧{exit}出口驶出，前往{destination}"},"slight right":{"default":"稍向右下匝道","name":"稍向右下匝道，驶入{way_name}","destination":"稍向右下匝道，前往{destination}","exit":"从右侧{exit}出口驶出","exit_destination":"从右侧{exit}出口驶出，前往{destination}"}},"on ramp":{"default":{"default":"上匝道","name":"上匝道，驶入{way_name}","destination":"上匝道，前往{destination}"},"left":{"default":"上左侧匝道","name":"上左侧匝道，驶入{way_name}","destination":"上左侧匝道，前往{destination}"},"right":{"default":"上右侧匝道","name":"上右侧匝道，驶入{way_name}","destination":"上右侧匝道，前往{destination}"},"sharp left":{"default":"急向左上匝道","name":"急向左上匝道，驶入{way_name}","destination":"急向左上匝道，前往{destination}"},"sharp right":{"default":"急向右上匝道","name":"急向右上匝道，驶入{way_name}","destination":"急向右上匝道，前往{destination}"},"slight left":{"default":"稍向左上匝道","name":"稍向左上匝道，驶入{way_name}","destination":"稍向左上匝道，前往{destination}"},"slight right":{"default":"稍向右上匝道","name":"稍向右上匝道，驶入{way_name}","destination":"稍向右上匝道，前往{destination}"}},"rotary":{"default":{"default":{"default":"进入环岛","name":"通过环岛后驶入{way_name}","destination":"通过环岛后前往{destination}"},"name":{"default":"进入{rotary_name}环岛","name":"通过{rotary_name}环岛后驶入{way_name}","destination":"通过{rotary_name}环岛后前往{destination}"},"exit":{"default":"进入环岛后从{exit_number}出口驶出","name":"进入环岛后从{exit_number}出口驶出，上{way_name}","destination":"进入环岛后从{exit_number}出口驶出，前往{destination}"},"name_exit":{"default":"进入{rotary_name}环岛后从{exit_number}出口驶出","name":"进入{rotary_name}环岛后从{exit_number}出口驶出，上{way_name}","destination":"进入{rotary_name}环岛后从{exit_number}出口驶出，前往{destination}"}}},"roundabout":{"default":{"exit":{"default":"进入环岛后从{exit_number}出口驶出","name":"进入环岛后从{exit_number}出口驶出，上{way_name}","destination":"进入环岛后从{exit_number}出口驶出，前往{destination}"},"default":{"default":"进入环岛","name":"通过环岛后驶入{way_name}","destination":"通过环岛后前往{destination}"}}},"roundabout turn":{"default":{"default":"{modifier}转弯","name":"{modifier}转弯，驶入{way_name}","destination":"{modifier}转弯，前往{destination}"},"left":{"default":"左转","name":"左转，驶入{way_name}","destination":"左转，前往{destination}"},"right":{"default":"右转","name":"右转，驶入{way_name}","destination":"右转，前往{destination}"},"straight":{"default":"继续直行","name":"继续直行，驶入{way_name}","destination":"继续直行，前往{destination}"}},"exit roundabout":{"default":{"default":"驶离环岛","name":"驶离环岛，驶入{way_name}","destination":"驶离环岛，前往{destination}"}},"exit rotary":{"default":{"default":"驶离环岛","name":"驶离环岛，驶入{way_name}","destination":"驶离环岛，前往{destination}"}},"turn":{"default":{"default":"{modifier}转弯","name":"{modifier}转弯，驶入{way_name}","destination":"{modifier}转弯，前往{destination}"},"left":{"default":"左转","name":"左转，驶入{way_name}","destination":"左转，前往{destination}"},"right":{"default":"右转","name":"右转，驶入{way_name}","destination":"右转，前往{destination}"},"straight":{"default":"直行","name":"直行，驶入{way_name}","destination":"直行，前往{destination}"}},"use lane":{"no_lanes":{"default":"继续直行"},"default":{"default":"{lane_instruction}"}}}}
```

Comment:
Is it necessary to check in all the translations from OSRM Text Instructions as test fixtures in this repository? Could OSRM Text Instructions be included as a submodule or NPM dependency?

Reply:
Included OSRMTI as a submodule.